### PR TITLE
always use memcpy to enforce alignment in getblock8

### PR DESCRIPTION
**Pull request by dimpase**
hopefully the compiler is clever enough to optimise this away

### CODE REVIEW

**Comment by dimpase** [comment]
Should I amend the commit to include the paragraph suggested above?
Perhaps it's easier that whoever merges this adds this into the commit message.

**Comment by fingolfin** [comment]
This comment really should be in the code, not the commit message (where it is invisible when reading the code)

**Comment by dimpase** [comment]
> This comment really should be in the code
OK, done.

File: src/intfuncs.c [hunk 1]
```diff
@@ -307,10 +307,18 @@ static inline uint64_t rotl64 ( uint64_t x, int8_t r )
 //-----------------------------------------------------------------------------
 // Block read - if your platform needs to do endian-swapping or can only
 // handle aligned reads, do the conversion here
+//
+// The pointer p may not be aligned, which means that directly writing to it can
```
Comment (ChrisJefferson) [inline src/intfuncs.c]: very picky, it's "reading it", not "writing to it"

Comment (dimpase) [reply src/intfuncs.c]: OK, fixed.

**Pull request merged**